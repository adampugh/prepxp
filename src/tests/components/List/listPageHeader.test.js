import React from "react";
import { shallow } from "enzyme";

import ListPageHeader from "../../../components/List/listPageHeader";

describe("<ListPageHeader />", () => {
    it("should match snapshot", () => {
        const wrapper = shallow(<ListPageHeader />);
        expect(wrapper).toMatchSnapshot();
    });
});