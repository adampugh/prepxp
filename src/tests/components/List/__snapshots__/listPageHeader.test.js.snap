// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListPageHeader /> should match snapshot 1`] = `
<div
  className="listPage__header"
>
  <div
    className="container-s"
  >
    <div
      className="listPage__header__content"
    >
      <div>
        <h1>
           
          <i
            className="fas fa-edit"
          />
        </h1>
      </div>
    </div>
  </div>
</div>
`;
